club/theme.json

```diff
@@ -46,6 +46,16 @@
 					"slug": "background",
 					"color": "#F6EB80",
 					"name": "Background"
+				},
+                {
+					"slug": "primary",
+					"color": "#000AFF",
```

Comment:
What is the benefit to having this twice (both foreground and primary)? I guess if the user wants to change the button and link colors for example but not the foreground color?